modelon/impact/client/entities/case.py

```diff
@@ -388,6 +403,23 @@ def initialize_from_case(self) -> Optional[Case]:
 
     @initialize_from_case.setter
     def initialize_from_case(self, case: Case) -> None:
+        """Sets the case to initialize from.
```

Comment:
Properties should be fully documented on "getters" (setter not visible in generated docs)